### ISSUE TITLE
Offer link to download latest release of Joomla

### ISSUE THREAD

**Issue by gitressa** [joomla/joomla-cms#37309]
Most open source CMS'es offer links for the latest release file. Maybe Joomla could also consider this, and share it on downloads.joomla.org?

**Drupal** - https://www.drupal.org/download
- https://www.drupal.org/download-latest/zip
- https://www.drupal.org/download-latest/tar.gz

**Wordpress** - https://wordpress.org/download/
- https://wordpress.org/latest.zip
- https://wordpress.org/latest.tar.gz

**Backdrop**
-  https://github.com/backdrop/backdrop/releases/latest/download/backdrop.zip

**Comment by HLeithner** [comment]
http://www.joomla.org/latest

Sadly this redirects to Joomla 3.10 this will be fixed with the release of Joomla 4.2

**Comment by gitressa** [comment]
Thanks. Is there a `tar.gz` version? I should have added it in the Issue summary, but the idea is to use the link in Lando, like this:

```
lando init \
  --source remote \
  --remote-url https://www.joomla.org/latest/tar.gz \
  --remote-options="--strip-components 1" \
  --recipe joomla \
  --webroot . \
  --name my-first-joomla-app
```

If you compare the Lando Quick Start commands, it's a constant task to update Joomla to the latest version, whereas with Drupal and WordPress, you automatically get the latest version:

- https://docs.lando.dev/joomla/getting-started.html#quick-start
- https://docs.lando.dev/wordpress/getting-started.html#quick-start
- https://docs.lando.dev/drupal/getting-started.html#quick-start
